honeybee_radiance/sensorgrid.py

```diff
@@ -23,7 +23,9 @@ class SensorGrid(object):
     Args:
         identifier: Text string for a unique SensorGrid ID. Must not contain spaces
             or special characters. This will be used to identify the object in the
-            exported Radiance files.
+            exported Radiance files. You may use / in name to identify grid groups.
+            For example floor_1/living_room create a sensor grid with living_room
+            identifier and floor_1 group identifier.
```

Comment:
Something this implicit is definitely ok for the Grasshopper interface but I was imagining that the core libraries would be more explicit about making you set the `group_identifier` if it's something that you want. For example, I'm not sure that I'm a fan of the idiosyncrasy that it creates in the dict schema because you can technically set the `group_identifier` two ways from a dictionary (one that explicitly uses a key and another that just includes a `/` in the identifier).

I can also see a case at some point where someone includes a `/` in their input for the sensorgrid identifier, they don't get an error as they usually would would for illegal radiance characters in IDs and so they think `/` is a legal radiance character (like it is in EnergyPlus), Then, they go to select sensorgrids from a model using the identifer that they originally input to the sensor grid here and they get nothing back because the actual grid identifier doesn't have a `/` in it.

I guess this just has a code smell to it.

Maybe it also helps to know that this implicit feature is not something that I am going to use on the Grasshopper interface because the generic methods that process the user's `_name_` input will clean out the `/` before assigning it as an identifier. So I'm going to have to write a small custom method for this case on the interface end anyway and I guess the only use case this serves is to save Python scripters a single line of code for having to explicitly set the group identifier. So I'm not really sure it's worth it.